Tests/EventListener/StartControllerSpanSubscriberTest.php

```diff
@@ -48,4 +48,25 @@ public function testOnController(): void
 
         $this->subject->onController($event->reveal());
     }
+
+    public function testOnControllerWithoutRoute(): void
```

Comment:
Did you just forget to adjust the test ? :)

Reply:
Yes, good catch. That's fixed.